spec/controllers/api/teams_controller_spec.rb

```diff
@@ -180,7 +180,10 @@
       expect(folder_relationships_length).to be(1)
     end
 
-    it 'returns encryptable files for team_id, in order from created_at' do
+    ## TODO: Activate test after order logic has been implemented again
+    # Has been ignored because order logic in serializer had to be removed to make search work
+    # Ticket: https://github.com/orgs/puzzle/projects/5/views/1?pane=issue&itemId=51495477
```

Comment:
use direkt github link without project board ....
this is not the follow up ticket for fixing this, is it?
better just link the follow up ticket, and inside the new ticket reference to the old one

Reply:
I changed the link to navigate directly to the follow up issue and referenced this ticket in the follow up story.

Reply:
how about a shorter comment:
TODO: Re-enable after fixing encryptables ordering, see #760